youtube_dl/extractor/videocdn.py

```diff
@@ -0,0 +1,72 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_str,
+    compat_urlparse,
+)
+
+from ..utils import determine_ext
+
+
+class VideoCdnIE(InfoExtractor):
+    _VALID_URL = r'https?://e\.video-cdn\.net/video?.*video-id=(?P<id>[a-zA-Z0-9-_]+).*'
+    _TESTS = [
+        {
+            'url': 'https://e.video-cdn.net/video?video-id=8eBUrWaMJFS38A5X-j2CgY&player-id=53Tun3ZZpZpVuvaTvsm3jU',
+            'info_dict': {
+                'id': '8eBUrWaMJFS38A5X-j2CgY',
+                'ext': 'mp4',
+                'title': 'RiskBuster FireFighter VI - Adventskranz',
+                'thumbnail': r're:(?i)https://.*\.jpeg',
+            },
+        },
+        {
+            'url': 'https://e.video-cdn.net/video?video-id=91imQ_wKjkTFghe-3mmBAA&player-id=7nCLZ_ESM8rT9YUw6qUGA9',
+            'info_dict': {
+                'id': '91imQ_wKjkTFghe-3mmBAA',
+                'ext': 'mp4',
+                'title': 'SCC2019_Talk_Tychsen_TXL.mp4',
+                'thumbnail': r're:(?i)https://.*\.jpeg',
+            },
+        },
+    ]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, video_id)
+
+        formats = []
+
+        video_id = self._search_regex(
```

Comment:
Is this different from the ID captured in the URL? If so, consider making one `display_id`. As you aren't using this one to get the media links, the search can/should have `default=None`.

Reply:
No, I will remove this search.